mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -668,6 +746,34 @@ impl UtxoRpcClientOps for NativeClient {
         )
     }
 
+    fn display_balances(&self, addresses: Vec<Address>, decimals: u8) -> UtxoRpcFut<Vec<(Address, BigDecimal)>> {
+        fn address_balance_from_unspent_map(address: &Address, unspent_map: &UnspentMap, decimals: u8) -> BigDecimal {
```

Comment:
If this can be reused somewhere else, it's preferred to move it outside another fn.

Reply:
I think `address_balance_from_unspent_map` is pinned to the `NativeClient::display_balances`, but I think this can be reused later somewhere:
```
unspents.iter().fold(BigDecimal::from(0), |sum, unspent| {
                sum + big_decimal_from_sat_unsigned(unspent.value, decimals)
            })
```

Reply:
Moved `address_balance_from_unspent_map` outside the `NativeClient::display_balances` method